### ISSUE TITLE
The Widget to Clear Cache is not working

### ISSUE THREAD

**Issue by 01Kuzma** [OFFLINE-GmbH/oc-responsive-images-plugin#97]
Hello,
Tested on localhost, PHP 8.1, Winter CMS.
Pressing `Clear Image Cache` button throws this error:

```
"Undefined constant "STDIN"" on line 109 of C:\laragon\www\winter3\vendor\symfony\console\Helper\QuestionHelper.php
```

**Comment by tobias-kuendig** [comment]
Are you on v2.5.5? This issue should have been fixed in #96

**Comment by 01Kuzma** [comment]
Well, yes, it was 2.5.4. And I'm on a Winter CMS, it's not updating to the latest version :(
